clusterloader2/testing/load/modules/reconcile-objects.yaml

```diff
@@ -0,0 +1,311 @@
+## Input params
+# Valid actions: "create", "scale and update", "delete"
+{{$actionName := printf "%s objects" .actionName}}
+{{$namespaces := .namespaces}}
+
+# Derivative variables
+{{$is_creating := (eq .actionName "create")}}
+{{$is_scaling := (eq .actionName "scale and update")}}
+{{$is_deleting := (eq .actionName "delete")}}
+{{$tuning_set := "N/A"}}
+{{if $is_creating}}{{$tuning_set = "RandomizedSaturationTimeLimited"}}{{end}}
+{{if $is_scaling}}{{$tuning_set = "RandomizedScalingTimeLimited"}}{{end}}
+{{if $is_deleting}}{{$tuning_set = "RandomizedDeletionTimeLimited"}}{{end}}
+{{$randomScaleFactor := DefaultParam .randomScaleFactor 0}}
+{{$minReplicaFactor := SubtractFloat 1 $randomScaleFactor}}
+{{$maxReplicaFactor := AddFloat 1 $randomScaleFactor}}
+
+# DaemonSets
+{{$daemonSetImage := DefaultParam .daemonSetImage "k8s.gcr.io/pause:3.0"}}
+{{$daemonSetReplicas := .daemonSetReplicas}}
+
+# Deployments
+{{$bigDeploymentSize := .bigDeploymentSize}}
+{{$bigDeploymentsPerNamespace := .bigDeploymentsPerNamespace}}
+{{$mediumDeploymentSize := .mediumDeploymentSize}}
+{{$mediumDeploymentsPerNamespace := .mediumDeploymentsPerNamespace}}
+{{$smallDeploymentSize := .smallDeploymentSize}}
+{{$smallDeploymentsPerNamespace := .smallDeploymentsPerNamespace}}
+
+# StatefulSets
+{{$smallStatefulSetSize := .smallStatefulSetSize}}
+{{$smallStatefulSetsPerNamespace := .smallStatefulSetsPerNamespace}}
+{{$mediumStatefulSetSize := .mediumStatefulSetSize}}
+{{$mediumStatefulSetsPerNamespace := .mediumStatefulSetsPerNamespace}}
+
+# Jobs
+{{$bigJobSize := .bigJobSize}}
+{{$bigJobsPerNamespace := .bigJobsPerNamespace}}
+{{$mediumJobSize := .mediumJobSize}}
+{{$mediumJobsPerNamespace := .mediumJobsPerNamespace}}
+{{$smallJobSize := .smallJobSize}}
+{{$smallJobsPerNamespace := .smallJobsPerNamespace}}
+
+## CL2 params
+{{$CHECK_IF_PODS_ARE_UPDATED := DefaultParam .CL2_CHECK_IF_PODS_ARE_UPDATED true}}
+{{$DISABLE_DAEMONSETS := DefaultParam .CL2_DISABLE_DAEMONSETS false}}
+{{$ENABLE_DNSTESTS := DefaultParam .CL2_ENABLE_DNSTESTS false}}
+{{$ENABLE_NETWORKPOLICIES := DefaultParam .CL2_ENABLE_NETWORKPOLICIES false}}
+{{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS true}}
+{{$SMALL_STATEFUL_SETS_PER_NAMESPACE := DefaultParam .SMALL_STATEFUL_SETS_PER_NAMESPACE 1}}
+{{$MEDIUM_STATEFUL_SETS_PER_NAMESPACE := DefaultParam .MEDIUM_STATEFUL_SETS_PER_NAMESPACE 1}}
```

Comment:
I think we should take this two values from "smallStatefulSetsPerNamespace" and "mediumStatefulSetsPerNamespace"

Reply:
Fixed